package.json

```diff
@@ -63,7 +65,17 @@
     "@babel/preset-react": "^7.12.7",
     "@babel/register": "^7.12.1",
     "@skyscanner/stylelint-config-skyscanner": "^6.0.0",
-    "autoprefixer": "^10.4.12",
```

Comment:
downgraded this (this is the same version we use on bpk web) as the latest version has some breaking changes which we did not account for when doing the upgrade as the config which uses this lib wasn't actually working